examples/mdetr/optimizer.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Copyright (c) Aishwarya Kamath & Nicolas Carion. Licensed under the Apache License 2.0. All Rights Reserved
```

Comment:
should we remove this line and add a comment with a link to the code ref?

Reply:
Yeah I plan to add a NOTICES file under the mdetr/examples root to cover everything in there